app/models/pipeline_run_stage.rb

```diff
@@ -2,6 +2,9 @@ class PipelineRunStage < ApplicationRecord
   include ApplicationHelper
   include PipelineOutputsHelper
   belongs_to :pipeline_run
+
+  PIPELINE_INSTALL_COMMAND = "pip install git+https://github.com/chanzuckerberg/idseq-pipeline.git".freeze
```

Comment:
You probably want `pip install --upgrade` here?   If ever reinstalling on same instance.

Reply:
This is fine assuming it only runs once on the instance/docker container, and there are no concurrent invocations.

Reply:
Thanks!